src/Models/ElevatedItem.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Firesphere\SolrSearch\Models;
+
+use SilverStripe\ORM\DataObject;
+
+class ElevatedItem extends DataObject
+{
+    private static $table_name = 'ElevatedItem';
+
+    private static $db = [
+        'Rank' => 'Int',
+        'Title' => 'Varchar(255)',
+        'ObjectClass' => 'Varchar(255)',
+        'ObjectID' => 'Int',
+        'SolrID' => 'Varchar(255)',
+        'Include' => 'Boolean(1)',
```

Comment:
For readability, I prefer using `Boolean(true)` and `Boolean(false)` over `1` and `0`